ocr_service/README.md

```
@@ -0,0 +1,100 @@
+```
+
+Request body:
+```json
+{
+  "pdf_url": "https://example.com/document.pdf"
+}
+```
+
+### Extract Text from Uploaded PDF File
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)